client/post-editor/editor-sharing/accordion.jsx

```diff
@@ -46,12 +43,9 @@ const EditorSharingAccordion = React.createClass( {
 		}
 
 		const skipped = PostMetadata.publicizeSkipped( post );
-		const targeted = connections.filter( ( connection ) => {
-			return ! includes( skipped, connection.keyring_connection_ID );
-		} );
-		const targetedServices = serviceConnections.getServicesFromConnections( targeted );
+		const targeted = filter( connections, ( { keyring_connection_ID: ID } ) => ! ( ID in skipped ) );
```

Comment:
- Proper camel-casing would be `id`
- `skipped` is an array, so we can't use the `in` operator unless we're checking for a particular index. Otherwise consider `-1 !== skipped.indexOf( id )` or `! includes( skipped, id )` (from Lodash), the latter being my preference.